### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by Stearnzy**
Co-authored-by: Brett Sherman <64513280+BJSherman80@users.noreply.github.com>
Co-authored-by: Christopher Allbritton <63476564+Callbritton@users.noreply.github.com>
Co-authored-by: Dani Coleman <60626984+dcoleman21@users.noreply.github.com>
Co-authored-by: Hanna Davis <63931068+Oxalisviolacea@users.noreply.github.com>
Co-authored-by: Hope Gochnour <52016184+hopesgit@users.noreply.github.com>
Co-authored-by: Jake Heft <60531761+jakeheft@users.noreply.github.com>
Co-authored-by: John Kim <65986168+abcdefghijohn@users.noreply.github.com>
Co-authored-by: Jose Lopez <61892154+JoseLopez235@users.noreply.github.com>
Co-authored-by: Kevin Cuadros <63522369+kevxo@users.noreply.github.com>

Description:  
* Documentation updated to include overview, design, setup, deploying on localhost, and authors with profile links.
* More will be added/changed over time, but here's a start. 

* [x] Updated README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,76 @@
...
+## Table of Contents
+  - [Design](#design)
+  - [Setup](#setup)
+  - [Running the Tests](#running-the-tests)
+  - [Deployment](#deployment)
+  - [Authors](#authors)
 
-* Database initialization
+### Design
+  * The project is split into three repositories, one for Front-End, one for Back-End, and one for a Sinatra Microservice.  This repository in particular is responsible for the user-facing views along with all services and controllers necessary to query our backend's API.
```
Comment (dcoleman21) [inline README.md]: We can add some key words here: split into three repositories, implementing a Service Oriented Architecture for future scalability. 
Or something like that??

**Review by dcoleman21 — CHANGES_REQUESTED** [review]
I made a suggestion. Please disregard if you do not feel this is necessary.

File: README.md (hunk 1 above)
Comment (Stearnzy) [reply README.md]: I love it, putting that in.

**Review by dcoleman21 — APPROVED** [review]
Looks good.

**Pull request merged**